ruler-cli/src/main/java/com/spotify/ruler_cli/Main.kt

```diff
@@ -0,0 +1,5 @@
+package com.spotify.ruler_cli
+
+class Main {
+    // TODO: Implement argument parsing logic.
```

Comment:
A place where the main method will parse the cli arguments and distribute them accordingly